slask.py

```diff
@@ -17,15 +17,15 @@
 import debt_utils
 from config import config
 
-def handle_message(message):
+def handle_message(command):
     # 'help'
```

Comment:
Sure I mean mainly I thought I needed to pass in text from another format so I was going the `get` outside the function.